src/plugins/intel_cpu/src/transformations/cpu_opset/common/pass/convert_matmul_to_fc.cpp

```diff
@@ -175,6 +175,8 @@ ov::intel_cpu::ConvertMatMulToFC::ConvertMatMulToFC() {
         auto fc = std::make_shared<ov::intel_cpu::FullyConnectedNode>(fc_input_a, fc_input_b, output_rank,
                 matmul->get_output_element_type(0));
         fc->set_friendly_name(matmul->get_friendly_name());
+        if (ov::fp16_compression_is_disabled(matmul))
```

Comment:
@itikhono Given we are now using `precise` rt_info not only for FP16, but for BF16 precision as well, shouldn't we rename all corressponding functions/attributes like `DisableFP16Compression` -> `DisablePrecisionLowering` or better name.

Reply:
@dmitry-gorokhov thanks for noticing this. I think better to do it as a separate activity. I opened ticket for that CVS-130863